core/commands/files/files.go

```diff
@@ -756,6 +794,69 @@ are run with the '--flush=false'.
 	},
 }
 
+var FilesUpdateCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Change the cid version of hash function of the root node of a given path.",
```

Comment:
s/of/or/

Also, given this description, maybe we should call the command `re-cid` ?

Reply:
How about `chcid`?